app/views/admin/publishers/index.html.slim

```diff
@@ -20,11 +20,17 @@ div.row
               th Name
               th Email
               th Status
+              th = sort_link(Publisher::VERIFIED_CHANNEL_COUNT, "Number of verified channels")
+              th = sort_link(:created_at, "Account creation time")
+              th = sort_link(:last_sign_in_at, "Last sign in at")
             tbody
-              - @publishers.each do |publisher|
+              - @publishers.left_joins(:channels).each do |publisher|
                 tr.gradeX
                   td = link_to publisher.id, admin_publisher_path(publisher.id)
                   td = publisher.name
                   td = publisher.email
                   td = publisher_status(publisher)
+                  td = publisher.channels.verified.count
+                  td = publisher.created_at
+                  td = publisher.last_sign_in_at
```

Comment:
I find dates with timezones kind of hard to read, what do you think about using [distance_of_times_in_words](https://apidock.com/rails/ActionView/Helpers/DateHelper/distance_of_time_in_words) and displaying their last sign in relative to the current time?